src/Subscriber.ts

```diff
@@ -234,11 +234,13 @@ class SafeSubscriber<T> extends Subscriber<T> {
     if (!this.isStopped) {
       const { _parentSubscriber } = this;
       if (this._complete) {
+        const wrappedComplete = () => this._complete.call(this._context);
```

Comment:
@trxcllnt what about this approach? create one fn closure for `complete` only, and let `__try...` avoid any branching (`if`, or rest operator...). There is overhead to create one additional closure, but only for completion and will not impact `next`.